sormas-e2e-tests/src/test/resources/features/sanity/web/EpidemiologicalDataCase.feature

```diff
@@ -57,6 +57,110 @@ Feature: Epidemiological data coverage
+    And I click on edit Exposure vision button
+    And I select Travel option in Type of activity from Combobox in Exposure form
+    Then I fill Location form for Type of place by chosen "FACILITY" options in Exposure for Epidemiological data
+    And I click on save button in Exposure for Epidemiological data tab in Cases
+    And I am checking all Location data in Exposure are saved and displayed
+    And I click on save button in Exposure for Epidemiological data tab in Cases
+    And I click on save button from Epidemiological Data
+    Then I click on Contacts with source case known with UNKNOWN option
+    And I click on Contacts with source case known with NO option
+    And I click on Contacts with source case known with YES option
+    And I click on save button from Epidemiological Data
+    And I check if Contacts of Source filed is available
+    And I click on the NEW CONTACT button in in Exposure for Epidemiological data tab in Cases
+    And I click on the CHOOSE CASE button in Create new contact form in Exposure for Epidemiological data tab in Cases
+    And I search and chose the last case uuid created via API in the CHOOSE CASE Contact window
+    And I click on SAVE button in create contact form
 
-
-
+  @issue=SORDEV-5523 @env_de
+  Scenario: Enter an exposure data in Case Directory for DE version
+    Given I log in with National User
+    And I click on the Cases button from navbar
+    And I click on the NEW CASE button
+    When I create a new case with specific data for DE version
+    And I check case created from created contact is correctly displayed on Edit Case page for DE
+    And I navigate to case person tab
+    And I navigate to epidemiological data tab in Edit case page
+    And I click on Exposure details known with NEIN option
+    And I click on Exposure details known with UNBEKANNT option
+    And I click on Exposure details known with JA option
+    Then I click on New Entry in Exposure Details Known in Cases directory
+    And I tick a Probable infection environmental box in Exposure for Epidemiological data tab in Cases
+    And I select from Combobox all options in Type of activity field in Exposure for Epidemiological data tab for Cases for DE version
+    Then I select a Type of activity Sonstiges option in Exposure for Epidemiological data tab in Cases
+    And I fill a Type of activity details field in Exposure for Epidemiological data tab in Cases
+    Then I select a Type of activity Versammlung option in Exposure for Epidemiological data tab in Cases
+    And I select from Combobox all Type of gathering in Exposure for Epidemiological data tab in Cases for DE version
+    And I select a type of gathering Sonstiges option from Combobox in Exposure for Epidemiological data tab in Cases
+    And I fill a type of gathering details in Exposure for Epidemiological data tab in Cases
+    And I fill Location form for Type of place field by "Unbekannt" options in Case directory for DE version
```

Comment:
We were instructed not to use a different language in step definitions - even as a parameter.
This applies to all german words below as well.